tests/e2e/vsan_stretched_cluster.go

```diff
@@ -3374,14 +3512,14 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 
 		for i := 0; i < 6; i++ {
 			framework.Logf("Creating pvc %v with reclaim policy Delete", i)
-			pvc, err := createPVC(ctx, client, namespace, nil, diskSize, scDelete, "")
+			pvc, err := createPVC(ctx, client, namespace, nil, diskSize, scDelete, accessMode)
 			pvclaimsWithDelete = append(pvclaimsWithDelete, pvc)
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}
 
 		for i := 0; i < 8; i++ {
 			framework.Logf("Creating pvc %v with reclaim policy Retain", i)
```

Comment:
same review comment here, think of writing a util or there are many older utils which is creating PVC on a scale, you can modify those utils and can use it according to your requirement